code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,773 @@
+	Those turrets are for keeping things <b>IN</b> as much as they are for keeping them out.<br><br>
+	Coordinate with the security checkpoint there to ensure every one of them gets basic access to our facilities for the duration of their stay.<br><br>
+	Issue any basic gear sh-<br>
+	"}
+
+/obj/item/paper/dsbssigma/note4
+	name = "paper - 'Faded Note'"
+	info = {"<br>
+	<b>To Private Simmons:</b><br><br>
+	First of all - let me apologize for not being able to welcome you to the team and show you the ropes in person.<br><br>
+	Away missions have been increasingly demanding as of late and I'm scheduled for departure right before your arrival.<br><br>
+	Regardless, you came as a recommended as a highly capable individual and should have no problem getting familiarized with our setup.<br>
+	Here are the basic instructions for running out turbine generators:<br><br>
+	<b>1:</b> Make sure the flow pumps from the O2 and Plasma chambers are enabled and pumping gas into the mixer.<br>
+	<b>2:</b> Configure the mixer's ratios and enable it (when in doubt - set it to 50/50).<br>
+	<b>3:</b> Note the gas flow meeter to ensure the gas is flowing and enable the pumps along the pipleline leading into the ignition chambers (there should be 2 pumps per side, 1 directly after the mixer and 1 right before the chamber).<br>
+	<b>4:</b> Ensuring that all doors are properly secured you may initiate the ignition by using the button next to the door controller.<br>
+	<b>5:</b> Open the exterior output blast doors so the turbine does not overheat.<br>
+	<b>6:</b> Initiate the turbine boot up procedure on the console and configure the SMES units to evenly match the output.<br><br>
+	Best of luck to you! With me out - you're in charge of keeping this place patched up and running.<br><br>
+	-<i>Quartermaster Lindsey.</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/note5
+	name = "paper - 'Stained Note'"
+	info = {"<br>
+	To whom ever is next on janitorial duty:<br><br>
+	The disposal piping under dormitories appears to have clogged again, if the complaints about the odd smells are anything to go about.<br><br>
+	Flush down some heavy duty cleaner and hope it clears it up so we don't have to bother the engineering staff again.<br><br>
+	-<i>Lindsey.</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/note6
+	name = "paper - 'Torn Report'"
+	info = {"<br>
+	<b>Warden's Report, Cell 6 Incident.</b><br><br>
+	<i>Somehow the prisoner in cell 6 managed to aquire a set of tools and kitchen utensils that they then used to, over the span of several-</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/note7
+	name = "paper - 'Torn Research Report'"
+	info = {"<br><br><br><br>
+	<i>-n further testing, <b>subject 701</b> appears to somehow developed a mutation rendering it highly resistant to viral pathogen. It has been locked up with other specimen, pending transport fo-</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/researchnotes
```

Comment:
Might be better as a subtype of /obj/item/paper/researchnotes

Reply:
The researchnotes object calls new() to randomize its current tech levels which is why I opted against it. 
But if you insist I could try and make it a subtype instead.

Reply:
I don't insist, I just point out that you can refactor /researchnotes so as to support a non-randomized subtype.

Reply:
Done.